geometry_msgs/Pose target_l
geometry_msgs/Pose target_r

---

bool success